src/www/ui/api/index.php

```diff
@@ -245,12 +248,12 @@ function (\Slim\Routing\RouteCollectorProxy $app) {
 
 ////////////////////////////ADMIN-USERS/////////////////////
 $app->group('/users',
-  function (\Slim\Routing\RouteCollectorProxy $app) {
-    $app->get('[/{id:\\d+}]', UserController::class . ':getUsers');
-    $app->put('[/{id:\\d+}]', UserController::class . ':updateUser');
-    $app->post('', UserController::class . ':addUser');
-    $app->delete('/{id:\\d+}', UserController::class . ':deleteUser');
+  function (\Slim\Routing\RouteCollectorProxy $app) use ($pattern) {
     $app->get('/self', UserController::class . ':getCurrentUser');
+    $app->get("[/{pathParam:$pattern]", UserController::class . ':getUsers');
+    $app->put("[/{pathParam:$pattern]", UserController::class . ':updateUser');
```

Comment:
And if you are working on the previous change, please remove the `[]` from this path as the user information is a must for this endpoint unlike the `:getUsers`.

```suggestion
    $app->put("/{pathParam:$pattern", UserController::class . ':updateUser');
```